### ISSUE TITLE
auto-play rule uses ambiguous definition "mechanism" [4c31df]

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules.github.io#1016]
The rule [https://act-rules.github.io/rules/4c31df](https://act-rules.github.io/rules/4c31df) uses the WCAG definition of "mechanism". This definition is somewhat ambiguous, and so should be replaced with something unambiguous.
